### DISCUSSION TITLE
1D Heat Transfer from the surface connection

### DISCUSSION THREAD

**Discussion by SXFire12** [firemodels/cfast#2073] [Q&A]
Dear All, 

I'm trying to perform a problem where the heat transfer is the main parameter of interest. I read in the user guide where it states that we have to add the same connections to get a fraction of unity with "AND". 
![imagen (1)](https://github.com/firemodels/cfast/assets/100284932/ccb0e1d1-15ba-493e-8bfb-cd87d093e7de)
And below is how I did it in CFAST. 
<img width="751" alt="Screenshot 2024-01-18 125037" src="https://github.com/firemodels/cfast/assets/100284932/36d96431-d708-4268-a16d-e8b0cc8c920b">

My question is, Is my step correct in defining the surface connections? Because I receive a note from CFAST "Warning: Horizontal heat transfer connections to compartment 1. Fraction of connected surface area is less than 1. Remaining fraction will be allocated to the outside."

To give a more details visualization, enclosed is my file. 

Thank you
[Three_Compartments.zip](https://github.com/firemodels/cfast/files/13976179/Three_Compartments.zip)

**Comment by drjfloyd** [comment]
In the image, the first line states that Comp C has a horizontal heat transfer connection to Comp B that is 25 % of the Comp C wall area. That leaves 75 % of the Comp C wall area that is unassigned. This unassigned area will will be to the ambient.

The middle two lines state that Comp B has horizontal heat transfer connections to C and A that are respectively 75 % and 25 % of the Comp B wall area.  This adds up to 100 % of the Comp B wall area.

The last line is like the first only Comp A has 25 % of its wall area that is unassigned.

**Comment by SXFire12** [comment]
Thanks, I've corrected it. 

I have another question, is there any validation of the heat transfer through conduction in CFAST? I tried to compare the results with the FDS, which gives an excessive temperature in the adjacent room and strange results for the room further from the fire source (negative temperature). 

![imagen (2)](https://github.com/firemodels/cfast/assets/100284932/c68d017b-7f34-4093-89f8-d3ec18b33fac)


<img width="728" alt="GIthubqa" src="https://github.com/firemodels/cfast/assets/100284932/f8c7f946-9d73-4172-b3af-cd7984dddf79">

**Comment by rmcdermo** [comment]
@gforney @preneke I looked at the CFAST manuals and I see conduction *through* walls discussed in the tech guide, but I do not find any direct verification for conduction through walls in the V&V guide.  If I am missing it, please let me know.

**Reply by rpeacoc** [reply]
Verification of heat transfer in CFAST is included in section 3.4 of the CFAST V&V guide. It is limited to the study of targets (which do use the same conduction routine as is used for the compartment surfaces).

I suspect the issue you are seeing comes from the intercompartment heat transfer. While it works for a single compartment connected to a fire room, it can fail when a third room is connected to the compartment that doesn't have a fire.

**Reply by rmcdermo** [reply]
@rpeacoc Thanks for your answer.  But I think this points to some work we need to get done.  We need room-to-room verification cases.  And I am very concerned to hear that we cannot do heat transfer with three interconnected rooms.  What is the technical limitation behind this?

**Reply by drjfloyd** [reply]
CFAST tracks upper layer wall and lower layer wall and those are not fixed areas. CFAST isn't partitioning a wall into vertical portions and then doing heat transfer based on some average condition for that wall segment.

**Reply by rpeacoc** [reply]
1. Verification of wall heat transfer would be a good addition.

2. We currently allow the user to specify an arbitrary set of connections between compartments. With more than a single connection, the simplifying assumption that the upper wall and lower wall can each be represented by a single temperature can fail. 

Consider a three room case connected in series with a fire in the one compartment on one end. In reality, the middle compartment has one hot wall that's going heat from the fire room and one cooler wall opposite that is losing heat to the third room. With the two zone assumption in CFAST, the average wall heat transfer can say that the second room is, on average, gaining energy from the other rooms. For the third room at the end away from the fire, this is non-physical and leads to the temperature in the third room decreasing.

While it would be possible to do each wall separately (which we do in radiation), an actual case could connect numerous compartments in an arbitrary geometry to a single compartment. At this point, the wall conduction approaches a CFD model rather than a simple zone model and is way beyond our intent for CFAST. 

We may just want to eliminate the intercompartment heat transfer.

**Reply by SXFire12** [reply]
Thanks all for the comprehensive discussion. I understand the current state of the CFAST now. Relying on CFAST has very strong benefits in terms of calculation time. Maybe in the future emphasizing the surface-to-surface connection might help other users as well
